man/systemd-cryptenroll.xml

```diff
@@ -299,6 +299,17 @@
         signatures likely will validate against pre-existing certificates.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--tpm2-with-pin=</option><replaceable>BOOL</replaceable></term>
+
+        <listitem><para>When enrolling a TPM2 device, controls whether to require the user to enter a PIN
+        when unlocking the volume in addition to PCR binding, based on TPM2 policy authentication. The pin
+        needs to be 4-32 characters long. Defaults to <literal>no</literal>. Note that incorrect PIN entry
+        when unlocking increments the dictionary attack lockout mechanism, and may lock out users for a
```

Comment:
maybe say that this is a mechanism of the TPM2 chip, and not of our tools here. otherwise people will look for a config option in our stuff to configure that.

Reply:
Good point. Maybe we should also point to the tpm-tss2 tools that can be used to change the settings?

Reply:
yeah, that can't hurt.